src/charts/c3/c3-chart.directive.js

```diff
@@ -69,17 +78,23 @@
     return {
       restrict: 'A',
       scope: {
-        config: '='
+        config: '=',
+        callback: '='
```

Comment:
Can we name this as getChartCallback or generateChartCallback? In the off case where we need to add another callback.